### ISSUE TITLE
detects virus

### ISSUE THREAD

**Issue by Araraura** [DarthTon/Xenos#64]
windows defender and malwarebytes detect this as a virus "Spyware.PasswordStealer"

**Comment by ezvision** [comment]
Ok? Its open source, go to windows defender or whatever and click on "allow on device".

**Comment by shavitush** [comment]
Obviously. It uses a memory hacking library and the injector supports manual mapping and even injection via a kernel driver. Malware can utilize the same features, it makes sense..

**Comment by MrSpaghatti** [comment]
Its a false positive... most AVs will detect anything hacking even if its not really a threat
